quam/utils/qua_types.py

```diff
@@ -0,0 +1,42 @@
+__all__ = [
+    "ScalarInt",
+    "ScalarFloat",
+    "ScalarBool",
+    "QuaScalarInt",
+    "QuaScalarFloat",
+    "QuaVariableInt",
+    "QuaVariableFloat",
+    "AmpValuesType",
+    "ChirpType",
+    "StreamType",
+    "_PulseAmp",
+]
+
+try:
+    from qm.qua._expressions import Scalar, QuaScalar, QuaVariable
```

Comment:
The purpose is to import from qm.qua.type_hints and to stop importing from places which start with `_`.

See this PR:
https://github.com/qm-labs/qm-qua-sdk/pull/686

Reply:
Eventually yes, but it's not in the current 1.2.2 version that people are using, so it wouldn't work yet, right? 
Once the PR is merged and a new QUA version is released, I can try the imports in three steps
1. qm.qua.type_hinting
2. qm.qua._expressions
3. qm.qua._dsl

wdyt?

Reply:
Yes, but I don't see a reason to merge this before, you can already test it on a local build of the new SDK (or an RC version) and then we'll merge it once ready (which should be this week)

Reply:
Fixed with new version